src/test/ui/impl-trait/bindings.stderr

```diff
@@ -0,0 +1,35 @@
+error[E0434]: can't capture dynamic environment in a fn item
```

Comment:
I agree, it could definitely be clearer. I figured out what it meant immediately, but probably only because I've been working on this for a while. Will open an issue now.